javascript.js

```diff
@@ -1,9 +1,242 @@
+    popupDescription:
+      "Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it 1960s with the releaLorem Ipsum is simply dummy text of the printing and when an unknown printer took a galley of type and scrambled it 1960s",
+  },
+  {
+    number: 1,
+    heading: 'Multi-Post Stories',
+    description:
+      'Experimental content creation feature that allows users to add to an existing story over the course of a day without spamming their friends.',
+    image: 'images/Snapshoot Portfolio2.png',
+    popupImage: 'images/popup2.png',
+    languages: ['html', 'css', 'javascript', 'Ruby', 'GitHub', 'Bootstrap'],
+    profession: ['FACEBOOK', 'Full Stack Dev', '2015'],
+    popupDescription:
+      "Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it 1960s with the releaLorem Ipsum is simply dummy text of the printing and when an unknown printer took a galley of type and scrambled it 1960s",
+  },
+  {
+    number: 2,
+    heading: 'Facebook 360',
+    description:
+      "Exploring the future of media in Facebook's first Virtual Reality app; a place to discover and enjoy 360 photos and videos on Gear VR.",
+    image: 'images/Snapshoot Portfolio3.png',
+    popupImage: 'images/popup3.png',
+    languages: ['html', 'css', 'javascript', 'Ruby', 'GitHub', 'Bootstrap'],
+    profession: ['FACEBOOK', 'Full Stack Dev', '2015'],
+    popupDescription:
+      "Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it 1960s with the releaLorem Ipsum is simply dummy text of the printing and when an unknown printer took a galley of type and scrambled it 1960s",
+  },
+  {
+    number: 3,
+    heading: 'Uber Navigation',
+    description:
+      'A smart assistant to make driving more safe, efficient, and fun by unlocking your most expensive computer: your car.',
+    image: 'images/Snapshoot Portfolio4.png',
+    popupImage: 'images/popup4.png',
+    languages: ['html', 'css', 'javascript', 'Ruby', 'GitHub', 'Bootstrap'],
+    profession: ['Uber', 'Lead Developer', '2018'],
+    popupDescription:
+      "Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it 1960s with the releaLorem Ipsum is simply dummy text of the printing and when an unknown printer took a galley of type and scrambled it 1960s",
+  },
+];
+
+function project() {
+  const a = document.getElementById('project-portfolio');
```

Comment:
- [optional] I observed that your project cards (cards 2 and 4) image are not placed properly according to the figma design.

## Your design
<img width="287" alt="Untitled" src="https://user-images.githubusercontent.com/103386183/225605437-eb0594bf-2ab0-4665-8fad-600bb4b27bb3.png">